templates/author/author.css

```diff
@@ -0,0 +1,94 @@
+/* AUTHOR PAGE STYLES */
+.author main{
+    font-family: "Graphik LC Web", sans-serif;
```

Comment:
you should be able to get rid of most if not all of these styles as they will inherit from default.

only thing here I think you might need is the padding bottom/top, but even that I believe will be applied to all pages and should already be there